src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -288,6 +312,23 @@ public void Dispose()
             }
         }
 
+        private string GetInvocationId()
+        {
+            var invocationId = Interlocked.Increment(ref _nextInvocationId) - 1;
```

Comment:
Feels wrong for this to be generated in the redis logic. Maybe we should change the interface to pass this as an argument along with the invocation.